### ISSUE TITLE
MD010 incorrectly flags tabs inside indented code blocks

### ISSUE THREAD

**Issue by cespare** [rvben/rumdl#630]
### rumdl version

0.1.94

### Installation method

Pre-built binary

### Bug description

`rumdl` flags tabs that are inside indented code blocks. It seems to ignore the same leading tabs when they are inside fenced code blocks. I think that it should ignore tabs inside all code blocks.

To be clear, I mean that if the indented code block is itself indented using *spaces*, then I think that further leading *tabs* should be ignored.

### Steps to reproduce

Here's an example markdown file:

`````
Foo bar

    for range 100 {
    	foo()
    }

This is a fenced

```
for range 100 {
	foo()
}
```
`````

Note that there is a tab character before `foo()` in both of the code blocks. Then:

```
▶ rumdl check --no-config --enable MD010 x.md
x.md:4:5: [MD010] Found tab for alignment, use spaces instead [*]

Issues: Found 1 issues in 1 file (2ms)
Run `rumdl fmt` to automatically fix 1 of the 1 issues
```

### Expected behavior

I think it should flag neither of the code blocks.

Alternatively, it could flag both of the code blocks. That would at least be consistent. If it did, I would probably ask for a configuration option to turn that off, like markdownlint's `code_blocks` config option for MD010 :)

### Actual behavior

MD010 flags only the indented code block and not the fenced one.

### Configuration

```toml
n/a
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in **v0.1.95**. MD010 now treats indented and fenced code blocks consistently and ignores tabs inside both by default. A new `code_blocks` option (default `false`) lets you opt back in to flagging tabs within code blocks if you want.

Your repro (tab inside an indented code block) no longer triggers MD010. Closing as released in v0.1.95.

**Comment by cespare** [comment]
Thanks @rvben!
